templates/projects/tasks_browse.html

```diff
@@ -2,7 +2,7 @@
 {% set active_page = "projects" %}
 {% set active_project  = "all" %}
 {% set active_link = "tasks" %}
-{% set section = _('Browse tasks') %}
+{% set section = _('Task List') if request.args['view'] == 'tasklist' else 'Browse Tasks'%} 
```

Comment:
Should we consider moving this into a variable for re-use, since we now reference the argument at many different locations in the code?

```
{% set is_task_list = request.args['view'] == 'tasklist' %}
{% set section = _('Task List') if is_task_list else 'Browse Tasks' %}
```

Also, updating all other references to this argument in the file. For example [here](https://github.com/bloomberg/pybossa-default-theme/blob/fcdea2a49ef5b117056a4259cd274c6aae95c241/templates/projects/tasks_browse.html#L19).

*Optional.*

Reply:
As I am looking through the code it seems like this conditional is tested throughout the page. Is it worthwhile consider if some of these conditions can be consolidated by view type (tasklist vs browse task) or is just moving it to a variable and updating the other conditionals sufficient?

Reply:
Ok just going with the variable change